client/modules/i18n/startup.js

```diff
@@ -9,6 +9,7 @@ import { Reaction } from "/client/api";
 import { Shops, Translations } from "/lib/collections";
 import * as Schemas from "/lib/collections/schemas";
 import i18next, { packageNamespaces, getLabelsFor, getMessagesFor, i18nextDep } from "./main";
+import {mergeDeep} from "/lib/api";
```

Comment:
This rule was removed in ESLint v1.0 and replaced by the object-curly-spacing and array-bracket-spacing rules.